### PR TITLE
added vehicle_screen, vehicle_state_list_item

### PR DESCRIPTION
**Pull request by stephanbcbauer**
added missing screens/widgets in actual code

### CODE REVIEW

File: lib/widget/vehicle_screen.dart [hunk 1]
```diff
@@ -2,10 +2,18 @@ import 'package:flutter/material.dart';
...
+import 'package:garage_app/widget/vehicle_list_item.dart';
+import 'package:garage_app/widget/vehicle_state_list_item.dart';
+
 class VehicleScreen extends StatelessWidget {
-  final VehicleState vehicleState;
-  final Vehicle vehicle;
+  static const _title = 'Vehicle';
+  static const _appBarBackgroundColor = Colors.blueGrey;
 
+  VehicleState vehicleState;
```
Comment (arbielsk) [inline lib/widget/vehicle_screen.dart:15]: please make this final

File: lib/widget/vehicle_screen.dart [hunk 2]
```diff
@@ -2,10 +2,18 @@ import 'package:flutter/material.dart';
...
+  Vehicle vehicle;
```
Comment (arbielsk) [inline lib/widget/vehicle_screen.dart:16]: please make this final

File: lib/widget/vehicle_list_item.dart [hunk 3]
```diff
@@ -26,6 +31,16 @@ class VehicleListItem extends StatelessWidget {
       child: InkWell(
         splashColor: _inkWellColor,
         highlightColor: _inkWellColor,
+         onTap: () => Navigator.of(context).push(MaterialPageRoute<Null>(
```
Comment (arbielsk) [inline lib/widget/vehicle_list_item.dart:34]: is this tested? otherwise please remove from the PR

**Comment by arbielsk** [comment]
will merge as soon as comments are resolved

File: lib/widget/vehicle_screen.dart (hunk 2 above)
Comment (stephanbcbauer) [reply lib/widget/vehicle_screen.dart]: done

File: lib/widget/vehicle_screen.dart (hunk 1 above)
Comment (stephanbcbauer) [reply lib/widget/vehicle_screen.dart]: done

**Review by arbielsk — APPROVED** [review]
LGTM

**Pull request merged**